index.html

```diff
@@ -4,7 +4,185 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./Components/alert.css" />
+    <link rel="stylesheet" href="./Components/avatar.css" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.0/css/all.min.css"
+      integrity="sha512-BnbUDfEUfV0Slx6TunuB042k9tuKe3xrD6q4mg5Ed72LTgzDIcLPxg6yI2gcMFRyomt+yJJxE+zJwNmxki6/RA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
     <title>Document</title>
   </head>
-  <body></body>
+  <body style="margin: 1rem">
+    <section class="component">
+      <!-- ALERT COMPONENT -->
+      <h1>Alert Component</h1>
+
+      <h3>Basic Alert</h3>
+
+      <div class="alert alert-primary">
+        This is a Primary Alert<i class="fas fa-times"></i>
+      </div>
+
+      <div class="alert alert-secondary">
+        This is a Secondary Alert<i class="fas fa-times"></i>
+      </div>
+
+      <div class="alert alert-success">
+        This is a Success Alert<i class="fas fa-times"></i>
+      </div>
+
+      <div class="alert alert-danger">
+        This is a Danger Alert<i class="fas fa-times"></i>
+      </div>
+
+      <div class="alert alert-warning">
+        This is a Warning Alert<i class="fas fa-times"></i>
+      </div>
+
+      <br />
+      <br />
+
+      <h3>Outline Alert</h3>
+
+      <div class="alert alert-outline-primary">
+        This is a Primary Outline Alert<i class="fas fa-times"></i>
+      </div>
+
+      <div class="alert alert-outline-secondary">
+        This is a Secondary Outline Alert<i class="fas fa-times"></i>
+      </div>
+
+      <div class="alert alert-outline-success">
+        This is a Success Outline Alert<i class="fas fa-times"></i>
+      </div>
+
+      <div class="alert alert-outline-danger">
+        This is a Danger Outline Alert<i class="fas fa-times"></i>
+      </div>
+
+      <div class="alert alert-outline-warning">
+        This is a Warning Outline Alert<i class="fas fa-times"></i>
+      </div>
+
+      <br />
+      <br />
+      <br />
```

Comment:
Don't use so many line breaks. You can add spacing.

Reply:
Thanks for the feedback. I will do it right away.